app/components/ui/tale-tabs-selector/component.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+
+export default Ember.Component.extend({
```

Comment:
We should be using import Component from '@ember/component';
and using it as export default Component.extend({
See this file as an example.

This satisfies the linter's new-module-imports rule. There are other places in the code that are still using Ember.Component that we've been replacing as we run across them.

Reply:
> See this file as an example.

Was there supposed to be a link here? In any case, I am happy to loop thru and update the few new components with the preferred pattern

Reply:
Oops it should have been [here](https://github.com/whole-tale/dashboard/blob/9b948c667ed4c7fdbe975c081db4191cc82802c2/app/components/ui/tale-instances/component.js#L9)